### PR TITLE
hw01 Donghyun Kang

### PR DESCRIPTION
**Pull request by Donghyun-Kang-Soc**
I had not used R Markdown but I figured out how to do it.

A hard point:
- I spent some time to figure out how I could preview the result of the the markdown documents in R Studio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+
+<img src="https://raw.githubusercontent.com/nerdizzyz/hw01/master/KANG_DONGHYUN.jpg">
+
+# Donghyun Kang's Biography
+
+My name is Donghyun Kang, a 1st-year Ph.D. student in the department of sociology. I was born in **Seoul**, South Korea on Sep 25, 1989. I did my B.A. at Seoul National University majoring in Business Administration and shifted to sociology for master's degree at the same university. For my master's thesis, I analyzed how the tweets regarding the two social movement events - *the LGBT group's sitting in the City Hall of Seoul* and *the protests of laid-off workers of Ssangyong Motor* - both of which happened in South Korea in 2014. I completed my master's degree in Feb 2016 and Before coming to Chicago in Aug 2017, I worked as a research associate at Social Network Computing Center at Seoul National University for one and a half of years. I'm currently interested in how and when the new academic disciplines emerge. More specifically, I hope to figure out the societal conditions that cause *interdisciplinary studies* to emerge. 
+
+* Research interests
+  Sociology of Knowledge; Computational Social Science; Economic Sociology; Social Organization
```
Comment (csava1025) [inline README.md:11]: This looks great! Something you could do next time is create a list for these "research interests."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
 # hw01
-Repository for homework 1
+Homework 1 from Donghyun Kang
+
+<img src="https://raw.githubusercontent.com/nerdizzyz/hw01/master/KANG_DONGHYUN.jpg">
```
Comment (csava1025) [inline README.md:4]: I was having trouble adding my own image  and ended up uploading it as a separate file. I didn't know about the "img src" function, so thank you! I will use it next time.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+
+# Donghyun Kang's Biography
+
+My name is Donghyun Kang, a 1st-year Ph.D. student in the department of sociology. I was born in **Seoul**, South Korea on Sep 25, 1989. I did my B.A. at Seoul National University majoring in Business Administration and shifted to sociology for master's degree at the same university. For my master's thesis, I analyzed how the tweets regarding the two social movement events - *the LGBT group's sitting in the City Hall of Seoul* and *the protests of laid-off workers of Ssangyong Motor* - both of which happened in South Korea in 2014. I completed my master's degree in Feb 2016 and Before coming to Chicago in Aug 2017, I worked as a research associate at Social Network Computing Center at Seoul National University for one and a half of years. I'm currently interested in how and when the new academic disciplines emerge. More specifically, I hope to figure out the societal conditions that cause *interdisciplinary studies* to emerge.
```
Comment (csava1025) [inline README.md:8]: This is awesome!

**Review by csava1025 — COMMENTED** [review]
Hi! This looked great. The only semi-feedback I had was that you could use a list next time.

File: README.md (hunk 3 above)
Comment (mobharmal) [inline README.md:8]: Successful coding for header, bold, and italics effects

File: README.md [hunk 4]
```diff
@@ -1,2 +1,14 @@
...
+
+# Donghyun Kang's Biography
+
+My name is Donghyun Kang, a 1st-year Ph.D. student in the department of sociology. I was born in **Seoul**, South Korea on Sep 25, 1989. I did my B.A. at Seoul National University majoring in Business Administration and shifted to sociology for master's degree at the same university. For my master's thesis, I analyzed how the tweets regarding the two social movement events - *the LGBT group's sitting in the City Hall of Seoul* and *the protests of laid-off workers of Ssangyong Motor* - both of which happened in South Korea in 2014. I completed my master's degree in Feb 2016 and Before coming to Chicago in Aug 2017, I worked as a research associate at Social Network Computing Center at Seoul National University for one and a half of years. I'm currently interested in how and when the new academic disciplines emerge. More specifically, I hope to figure out the societal conditions that cause *interdisciplinary studies* to emerge. 
+
+* Research interests
+  Sociology of Knowledge; Computational Social Science; Economic Sociology; Social Organization
+  
+Facebook
+[click](https://www.facebook.com/donghyun.kang)
```
Comment (mobharmal) [inline README.md:14]: Correct Markdown syntax for adding a link, nice work!

File: README.md (hunk 2 above)
Comment (mobharmal) [inline README.md:4]: Could have embedded an image using the following Markdown syntax:

File: README.md (hunk 1 above)
Comment (mobharmal) [inline README.md:11]: You already fulfill the requirements for the assignment, but could have put these research interests in a list if you wanted

**Review by mobharmal — COMMENTED** [review]
Evaluation HW-01 for Donghyun Kang
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |  Yes            |            |
| **Coding strategy**         |           |       Yes       |            |
| **Presentation**            |           |        Yes      |            |
| **Achievement, creativity** |           |   Yes           |            |
| **Ease of access**          |           |              |     Yes       |

Unable to see the full html file when I click the "view" button

File: README.md (hunk 2 above)
Comment (Donghyun-Kang-Soc) [reply README.md:4]: thanks!

**Review by nnickels — COMMENTED** [review]
Evaluation HW-01 for Donghyun Kang
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       X       |            |
| **Coding strategy**         |     X     |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |          |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Nice work on homework 1! You display use of several markdown elements and produce a solid README. 
* Unsure from your code if you were trying to create a list with your research interests, but using one would help presentation.
* Can also imbed image by saving it in the local repo and calling its file path from there.
* Good job figuring out how to preview with RStudio! 
* Your README appears completely find with using "View" of the .md file; no worries here.

**Pull request closed without merging**